helm/athena/templates/fixjob/fixjob-np.yaml

```diff
@@ -1,4 +1,33 @@
 {{- if and (not .Values.services.athena.letsencrypt) .Release.IsUpgrade .Values.fixJob.enabled }}
```

Comment:
Is there some weird indentation going on in this file or is GitHub showing it weird?